src/sass/core/letter-spacing.scss

```diff
@@ -3,45 +3,24 @@
 @use "../mixins";
 @use "../utils";
 
-// TODO: Change to `percent-range`.
-//
-// prettier-ignore
-$-range: (
-	-0.1,
-	-0.09,
-	-0.08,
-	-0.075,
-	-0.07,
-	-0.06,
-	-0.05,
-	-0.04,
-	-0.03,
-	-0.025,
-	-0.02,
-	-0.01,
-	 0,
-	 0.01,
-	 0.02,
-	 0.025,
-	 0.03,
-	 0.04,
-	 0.05,
-	 0.06,
-	 0.07,
-	 0.075,
-	 0.08,
-	 0.09,
-	 0.1,
-);
-
 // NOTE: Do not use `em($rv)`.
 //
 // - 0.1 + em -> 0.01em
 // - em(0.1)  -> 0.00625em
 //
 @mixin letter-spacing($variants...) {
 	@include mixins.compose($variants) using ($composed) {
-		@each $rv in $-range {
+		@each $rv in get.decimal-range(0, -0.1) {
```

Comment:
I see what you did here. There’s a small utility you can use to combine these loops. So you can use `utils.list-of` (I think that’s what it’s called, you can double-check the `utils` folder). It will create a new list from two separate lists. So you should be able to refactor this to be like: `utils.list-of(get.decimal-range(0, -0.1), get.decimal-range(0, 0.1))` and that will enable you to only have to have one loop instead of two.

The `get.n()` function just tells you in a number is negative or positive. So if we have separate loops, we have to have `get.n()` for the negative loop. So you don’t to worry about negative values when you combine loops. That is handled automatically.